tempest2/src/main/kotlin/app/cash/tempest2/AsyncView.kt

```diff
@@ -29,6 +30,8 @@ interface AsyncView<K : Any, I : Any> {
    */
   suspend fun load(key: K, consistentReads: Boolean = false): I? = loadAsync(key, consistentReads).await()
 
+  suspend fun loadWithCapacity(key: K, consistentReads: Boolean = false, returnConsumedCapacity: ReturnConsumedCapacity): ResultWithCapacityConsumed<I?> = loadAsyncWithCapacity(key, consistentReads, returnConsumedCapacity).await()
```

Comment:
Not super happy with the method name.

Reply:
Neither am I, I tried changing it to an overload.